src/os/web.rs

```diff
@@ -194,7 +208,7 @@ impl Target for Os {
             // Platform::Dive,
             // Platform::Fuchsia,
             // Platform::Redox,
-            Platform::Unknown(string.to_string())
+            Platform::Unknown(format!("Unrecognized platform: {}", string))
```

Comment:
This is redundant, as `Platform::Unknown` implies that the platform is unrecognized to whoami

Reply:
Yeah, just line 211.  Everything else is good!